feat: Wallet account view 4

janmichek opened the pull request:
<!--- Ensure the title of the Pull Request follows conventional commits syntax. If it resolves an issue, provide its ID in the scope section. -->

## Description
fixes #536 

There are 2 minor UX issues I will try to resolve in the following (Portfolio) issue, so I don't block the review

1. After successful wallet connection there is no loader when redirecting to connected account detail
2. When clicking on Try again, the whole page gets reloaded, which is not necessary. Only scanning again should work, but for some reason it's not working https://github.com/aeternity/aescan/issues/572

## Demo
<!--- Show us a video or screenshots that present the changes. Before/after comparison is very welcome -->

https://github.com/aeternity/aescan/assets/15363559/64887a9e-5d48-4f4d-ad37-92793c2700ff


## Checklist:
<!--- Put an `x` in all the boxes that apply. -->
<!--- If your change requires a documentation PR, please link it appropriately -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add wallet account detail

michele-franchi reviewed (changes requested):
Generally it looks good but I would adjust the following:
1. The "Connect Wallet" button should have the same height of the network switcher:
<img width="383" alt="image" src="https://github.com/aeternity/aescan/assets/11598532/72da2c10-dd57-4554-a909-142f1cef5bbd">

`padding: var(--space-1);` on the button solves it.

2. The "Exit Wallet" dropdown should be visually similar to the menu dropdowns.
<img width="200" alt="image" src="https://github.com/aeternity/aescan/assets/11598532/8657523e-1d30-4248-9328-3a78b63b8225">
Example:
<img width="233" alt="image" src="https://github.com/aeternity/aescan/assets/11598532/5382349b-c0b4-4cf1-8f8c-1525dbc55885">
Without the min-width of the other menu items.

3. When you connect the wallet and you refresh the page the connection is lost.

janmichek commented on .env.example:
```diff
@@ -9,6 +9,7 @@ NUXT_PUBLIC_NETWORK_NAME=MAINNET
 NUXT_PUBLIC_NETWORK_ID=ae_mainnet
 NUXT_PUBLIC_ALTERNATIVE_NETWORK_URL=http://localhost:8081
 NUXT_PUBLIC_ALTERNATIVE_NETWORK_NAME=TESTNET
+NUXT_PUBLIC_NETWORK_ID="ae_mainnet"
```
removed

janmichek commented:
> Generally it looks good but I would adjust the following:
> 
>     1. The "Connect Wallet" button should have the same height of the network switcher:
> 
> 
> <img alt="image" width="383" src="https://private-user-images.githubusercontent.com/11598532/313969491-72da2c10-dd57-4554-a909-142f1cef5bbd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNDgwMjIsIm5iZiI6MTcxMTM0NzcyMiwicGF0aCI6Ii8xMTU5ODUzMi8zMTM5Njk0OTEtNzJkYTJjMTAtZGQ1Ny00NTU0LWE5MDktMTQyZjFjZWY1YmJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDA2MjIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlM2I4Y2E4MGNiMDI5NWY3MzM1NjUzNjMwMTM2NjQ1MTNmOGYyYjIzY2I0MGYxY2U1ODg3NGNkOWZiMDkwYjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7Z_Lf3a-PWBcR1cKKWd8nKCPACEW9VJCHxTfGspMWd0">
> 
> `padding: var(--space-1);` on the button solves it.
> 
>     2. The "Exit Wallet" dropdown should be visually similar to the menu dropdowns.
> 
> 
> <img alt="image" width="200" src="https://private-user-images.githubusercontent.com/11598532/313970005-8657523e-1d30-4248-9328-3a78b63b8225.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNDgwMjIsIm5iZiI6MTcxMTM0NzcyMiwicGF0aCI6Ii8xMTU5ODUzMi8zMTM5NzAwMDUtODY1NzUyM2UtMWQzMC00MjQ4LTkzMjgtM2E3OGI2M2I4MjI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDA2MjIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1OWRhNzJmMjMwODNlMzI4ZTkxNTI1ZTZkNmQ5MTZhZTU1OWNhZGZlNjQ1MjVkNjBiZWRmYTNjZjMwODQzZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l5BDp6Tud5bVdVAturA2wKZKmvsq-GEgfHVI7ulUzHY">
> 
> Example:
> <img alt="image" width="233" src="https://private-user-images.githubusercontent.com/11598532/313970573-5382349b-c0b4-4cf1-8f8c-1525dbc55885.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNDgwMjIsIm5iZiI6MTcxMTM0NzcyMiwicGF0aCI6Ii8xMTU5ODUzMi8zMTM5NzA1NzMtNTM4MjM0OWItYzBiNC00Y2YxLThmOGMtMTUyNWRiYzU1ODg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDA2MjIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk2ODQ5Nzg2ZjMzYmRlNjFhNzFjMDBmZjA1NDYyZWQ4MTk5OTMzM2Y2MDc2MmIxM2UwZmZmODM4ZDVjYjRlNWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qryCKHcnIybJqQXP_JA2c3SCc3P2okIScnbanAwVWMU">
> 
> Without the min-width of the other menu items.
> 
>     3. When you connect the wallet and you refresh the page the connection is lost.

I have been discussing this with Denis. Currently is not possible to achieve this seamlessly. The app can remember last used account, but we cannot go over the user prompt to connect to the wallet again like on superhero.com
Implementation-wise, the SDK can do something to simplify it, but fundamentally will stay the same.

For now, I suggest not to implement it, wait for SDK completion, and the implement the persistency of last logged address @michele-franchi

Liubov-crypto commented:
1. Fixed. 
2. 
I found the reason behind the images blocking: 

![icn](https://github.com/user-attachments/assets/e9665490-aaf5-4d91-a2cd-4155cbb2f374)
![img](https://github.com/user-attachments/assets/73d791b9-6318-4d10-99c2-1eb0e681b02c)

janmichek commented:
> 1. Fixed.
> 
> 
> I found the reason behind the images blocking:
> 
> ![icn](https://private-user-images.githubusercontent.com/69896204/351271637-e9665490-aaf5-4d91-a2cd-4155cbb2f374.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3NDE4NzIsIm5iZiI6MTcyMTc0MTU3MiwicGF0aCI6Ii82OTg5NjIwNC8zNTEyNzE2MzctZTk2NjU0OTAtYWFmNS00ZDkxLWEyY2QtNDE1NWNiYjJmMzc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIzVDEzMzI1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1NDlmN2JjNWM2NWExZmUyZTFiMDhiMzIxZDliOTFkOWUyNDY1Y2NiODlhMDEzNGZjMWViNDE0MGQwNzIxMTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1Tyxcju4g7TSXOsA_vFxvqC-JAGs_-7HxobvP0fkdPE) ![img](https://private-user-images.githubusercontent.com/69896204/351271670-73d791b9-6318-4d10-99c2-1eb0e681b02c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3NDE4NzIsIm5iZiI6MTcyMTc0MTU3MiwicGF0aCI6Ii82OTg5NjIwNC8zNTEyNzE2NzAtNzNkNzkxYjktNjMxOC00ZDEwLTk5YzItMWViMGU2ODFiMDJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIzVDEzMzI1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxZmQyMWNlYTVmYTgzOGRiMGZiOGM1YTMxY2I5NDBkZTUwMDQxMmQxYjMwMDVhYzcwMjgwMjE5YWYzZjYxODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kEIuhO0Dyjn4fcjEY7ORV4xkpqqleIs6I8eDBEoI7UU)

Thanks, found the issue and fixed it.
